src/all/nijiero/src/eu/kanade/tachiyomi/extension/all/nijiero/NijieroTags.kt

```diff
@@ -0,0 +1,218 @@
+package eu.kanade.tachiyomi.extension.all.nijiero
+
+public val nijieroTags = arrayOf(
```

Comment:
Does this list update often? Consider updating it dynamically.

Reply:
not really sure if they really added anything... should i make it fetch the tags in some way? but where and how?

Reply:
Seeing as the site hasn't been updated since 2023, I think it should be fine to leave as-is. If you wanted to implement it you could either [piggyback off existing responses](https://github.com/keiyoushi/extensions-source/blob/de5d088129b326b37b2c95031ed36db02f1a10e8/lib-multisrc/mccms/src/eu/kanade/tachiyomi/multisrc/mccms/MCCMSWeb.kt#L38), or [fetch them specifically](https://github.com/keiyoushi/extensions-source/blob/de5d088129b326b37b2c95031ed36db02f1a10e8/lib-multisrc/madara/src/eu/kanade/tachiyomi/multisrc/madara/Madara.kt#L1053)

Reply:
the contents themselves are rather updated slowly, aswell. the last release is from a few months ago...